nextclade/defaults/config.yaml

```diff
@@ -10,7 +10,7 @@ metadata_url: "https://data.nextstrain.org/files/workflows/mumps/metadata.tsv.zs
 strain_id_field: "accession"
 
 clade_membership:
-  metadata: "defaults/{build}/reference_strains.tsv"
+  metadata: "{build}/reference_strains.tsv"
```

Comment:
Ditto `{build}` being unclear in the config.

The nextclade `builds` defined in the Snakefile should probably also be moved to a config param.

https://github.com/nextstrain/mumps/blob/41654ec8e601f4669146d4c3ac1834ae7f74c12f/nextclade/Snakefile#L11